HW0/P4/P4.py

```diff
@@ -51,12 +58,31 @@
     # A = ?
     # a = ?
     # s = ?
-
+    
+    A = np.array( [ [1,0,0, dt,  0,  0], 
```

Comment:
I agree! Another way to do it would be to create a matrix of zeros and set different indices to have certain values. You can just write the variable name and the index of the variable you would like to change.